src/components/common/Problem.vue

```diff
@@ -29,34 +30,107 @@ export default defineComponent({
+					el.async = true;
+					el.src = src;
+					shouldAppend = true;
+				} else if (/\.css(?:\??[0-9a-zA-Z=]*)$/.exec(src)) {
+					el = document.querySelector('link[href="' + src + '"]');
+					if (el && el.hasAttribute('data-loaded')) {
+						console.log(`${src} seems to have already been loaded!`);
+						resolve();
+						return;
+					} else if (el) {
+						break create_el;
+					}
+
+					el = document.createElement('link');
+					el.type = 'text/css';
+					el.rel = 'stylesheet';
+					el.href = src;
+					shouldAppend = true;
+				} else {
+					console.error(`Received invalid src: ${src}`);
+					reject();
+					return;
+				}
+
+				el.onerror = reject;
+				el.onabort = reject;
+				el.onload = function loadScriptHandler() {
+					console.log(`[onLoad] ${src} successfully loaded!`);
+					el?.setAttribute('data-loaded', 'true');
+					resolve();
+				};
+
+				if (shouldAppend) document.head.appendChild(el);
+			});
+		}
+
 		async function loadProblem() {
 			const formData = new FormData();
-			formData.set('problemSeed', '1');
+			formData.set('problemSeed', '12345');
 			formData.set('sourceFilePath', _file.value);
-			formData.set('outputFormat', 'static');
+			formData.set('outputFormat', 'ww3');
 
-			let value;
+			let value, js, css;
 			try {
 				const response = await axios.post('/renderer/render-api', formData,
 					{ headers: { 'Content-Type': 'multipart/form-data' } });
 
-				value = (response.data as { renderedHTML: string }).renderedHTML;
+				const data = response.data as RendererResponse;
+				console.log(data);
+
+				value = data.renderedHTML;
+				js = data.resources.js ?? [];
+				css = data.resources.css ?? [];
 			} catch(e) {
 				return;
 			}
 
 			if (!value) return;
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call
+			await Promise.all(js.map(
+				async (jsSource) => {
+					await loadScript(jsSource).
+						// then(() => console.log(`loaded ${jsSource}`)).
+						catch(() => console.error(`Could not load ${jsSource}`));
+				}
+			));
 
-			// Extract only the form element.
-			const match = /<form(.*)>(.*)<\/form>/s.exec(value);
-			if (match) html.value = match[0];
+			await Promise.all(css.map(
+				async (jsSource) => {
+					await loadScript(jsSource).
+						// then(() => console.log(`loaded ${jsSource}`)).
+						catch(() => console.error(`Could not load ${jsSource}`));
+				}
+			));
 
+			console.log('All js and css have been loaded...');
+			html.value = value;
 			/* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */
 			if (window.MathJax && typeof window.MathJax.typesetPromise == 'function') {
 				// eslint-disable-next-line @typescript-eslint/no-unsafe-return
 				window.MathJax.startup.promise.then(() => window.MathJax.typesetPromise([renderDiv.value]));
 			}
-			/* eslint-enable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call */
```

Comment:
This line should be added back.  Otherwise these rules are disabled for the rest of the file instead of only for that code block as was the intent.